ftp_s3/S3Authorizer.py

```diff
@@ -1,20 +1,19 @@
 import boto
+
 from pyftpdlib.authorizers import DummyAuthorizer, AuthenticationFailed
+from . import settings
 
 class S3Authorizer(DummyAuthorizer):
     def __init__(self, *args, **kwargs):
         DummyAuthorizer.__init__(self, *args, **kwargs)
         self.conn = None
 
-    def validate_authentication(self, aws_access_key_id, aws_secret_access_key, handler):
-        self.conn = boto.connect_s3(aws_access_key_id, aws_secret_access_key)
-
-        try:
-            self.conn.get_all_buckets()
-        except boto.exception.S3ResponseError:
+    def validate_authentication(self, username, password, handler):
+        if username == settings.FTP_USERNAME and password == settings.FTP_PASSWORD:
```

Comment:
Thanks @alaayameen, I don't think it will work this way because this should respond with FTP standers.